dashboard/cron.py

```diff
@@ -508,10 +508,10 @@ def submission(self):
             submission_url = f"""with sub_fact as (select submission_id, assignment_id, course_id, user_id, global_canvas_id, published_score from submission_fact sf join user_dim u on sf.user_id = u.id where course_id = '{data_warehouse_course_id}'),
                 enrollment as (select  distinct(user_id) from enrollment_dim where course_id = '{data_warehouse_course_id}' and workflow_state='active' and type = 'StudentEnrollment'),
                 sub_with_enroll as (select sf.* from sub_fact sf join enrollment e on e.user_id = sf.user_id),
-                submission_time as (select sd.id, sd.submitted_at, sd.graded_at, sd.posted_at at time zone 'utc' at time zone '{settings.TIME_ZONE}' as grade_posted_local_date from submission_dim sd join sub_fact suf on sd.id=suf.submission_id where (posted_at is not null and posted_at < getdate()) OR (posted_at is null and grade is null)),
```

Comment:
I'm glad this is removed because the use of `getdate()` didn't look stable.  That is, assuming the function returns the current date, each time the query is run, it would get a more recent date.  The later the code is run, that condition will eventually pass.  I don't think that was desired here.

Reply:
Good point!